### PR TITLE
Add contributors section

### PR DESCRIPTION
**Pull request by shivam-sharma7**
Add contributors section

### CODE REVIEW

File: src/assets/data/contributors.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    -Nasis
+    -Omiliades
+    -Patsouras
+    -Fakidis
+
+*/
+
+import i18n from "../../i18n";
+
+const Contributors = [
```
Comment (GeorgeApos) [inline src/assets/data/contributors.js:41]: As i told you in the issue, we need to take these data from the GitHub API :

- https://contrib.rocks/image?repo=Open-Source-UoM/MyUoM

or 

- https://stackoverflow.com/questions/48874412/github-api-how-to-get-the-top-contributors-sorted-for-a-given-repository

**Review by GeorgeApos — CHANGES_REQUESTED** [review]

File: src/assets/data/contributors.js (hunk 1 above)
Comment (shivam-sharma7) [reply src/assets/data/contributors.js:41]: @GeorgeApos How can we achieve the following style?

image


or you want this style?

Comment (GeorgeApos) [reply src/assets/data/contributors.js:41]: I like the style , however we can't support updating the Contributors data so we wan't to retrieve them through the Github API

Comment (shivam-sharma7) [reply src/assets/data/contributors.js:41]: @GeorgeApos I was trying to fetch contributors then I got the following error while fetching

image

Comment (shivam-sharma7) [reply src/assets/data/contributors.js:41]: @GeorgeApos I consumed Github API to display Contributors list

**Comment by papsavas** [comment]
Should we also filter out bots (`"type": "Bot"`)?

**Comment by shivam-sharma7** [comment]
> Should we also filter out bots (`"type": "Bot"`)?

 @GeorgeApos I don't have time to do this now and you can merge this, @papsavas raise a new issue for this

**Review by GeorgeApos — APPROVED** [review]

**Comment by GeorgeApos** [comment]
I like the bot idea. Raise a new issue.

**Comment by shivam-sharma7** [comment]
Thanks

**Pull request merged**